### PR TITLE
add s5p data block documentation

### PR DESCRIPTION
**Pull request by marooned20**
Pull Request
============

> Tracked in JIRA by [UP-5221](https://up42.atlassian.net/browse/UP-5221)
> Paired with @nekhtiari 

### Scope of the PR

The scope of PR is to submit documentation for new Sentinel-5p level-3 block.

**NOTE:**
Currently, the block is not published on production, therefore the marketplace url is incorrect. Once the block goes to prod, will update with the correct URL.

### Checklist

- [x] Checked spelling and grammar
- [x] Provided code samples where relevant
- [x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

File: source/up42-blocks/data/esa-s5p-l3-gtiff.rst [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  **or** ``intersects``.
+* ``intersects``: A GeoJSON geometry to use as an AOI. Will return all scenes that intersect with this geometry. Use
+  only ``intersects`` **or** ``bbox`` **or** ``contains``.
+* ``contains``: A GeoJSON geometry to use as an AOI. Will return all scenes that completely cover this geometry. Use only ``contains``
+  **or** ``intersects`` **or** ``bbox``
+* ``time``: A date range to filter scenes on. This range applies to the acquisition date/time of the scenes. Currently, date range is restricted to 31 days due to potential high throughput.
+* ``fail_on_missing``: A boolean flag to determine whether to fail or continue in case of images are not present for some dates. Defaults to ``False``.
+* ``layer``: A string describing layer type that is offered by Sentinel-5P. It **must** be one of the abbreviations listed above.
+* ``geojson_url``: A URL to geojson file with geometries. Must be accessible via simple HTTP call.
+* ``qa_mask``: A string describing which mask to use. This enable to filter pixel values based on quantile. Available masks are ``['nomask', 'mask50', 'mask75']``.
```
Comment (thaisbendixen) [inline source/up42-blocks/data/esa-s5p-l3-gtiff.rst:42]: "This enable**s** to filter pixel values based on quantile."

File: source/up42-blocks/data/esa-s5p-l3-gtiff.rst [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+Output format
+-------------
+The block outputs clipped Cloud Optimized GeoTIFFs for a given geometry, one image per day for a given date range. 
+When a url to geojson containing multiple geometries is provided, the output then is organized in sub-folders per geometry.
```
Comment (thaisbendixen) [inline source/up42-blocks/data/esa-s5p-l3-gtiff.rst:47]: "the output **is then** organized in sub-folders per geometry." <- i think? not sure

**Review by thaisbendixen — APPROVED** [review]
LGTM! Don't forget to add the query filters here https://docs.up42.com/going-further/filters.html?highlight=capabilities

**Review by teoEnache — APPROVED** [review]

**Review by markusuwe — APPROVED** [review]
LGTM

**Pull request merged**